webapp/src/tutorial.tsx

```diff
@@ -433,12 +434,19 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
         }
     }
 
+    private handleResize() {
+        const options = this.props.parent.state.tutorialOptions;
+        this.setShowSeeMore(options.autoexpandStep);
+    }
+
     componentDidMount() {
         this.setShowSeeMore(this.props.parent.state.tutorialOptions.autoexpandStep);
+        window.addEventListener('resize', pxt.Util.debounce(this.handleResize, 500));
     }
 
     onMarkdownDidRender() {
         this.setShowSeeMore(this.props.parent.state.tutorialOptions.autoexpandStep);
+        window.removeEventListener('resize', pxt.Util.debounce(this.handleResize, 500));
```

Comment:
debounce creates a new function instance so this won't remove the one added in componentDidMount. Store the debounce function in componentDidMount and remove that one.

Reply:
ahhh yeah that makes sense. Thanks, Peli!